version.py

```diff
@@ -0,0 +1,22 @@
+"""
+Copied from https://github.com/HannahVMeyer/limmbo
+
+Allows this package to be set up without having already been set up!
+"""
+import re
+from os.path import join
+from setuptools import find_packages
+
+
+def get():
+    """Returns the current version without importing pymds."""
+    pkgnames = find_packages()
+    if len(pkgnames) == 0:
+        return "unknown"
```

Comment:
I would have this blow up at this point, not silently return an invalid version. And same for the below. If a version string can't be found in the `__init__.py` file you should tell the user.  `raise ValueError` or something.

Reply:
Seems sensible. I confess I hadn't even really looked at what this code did, other than seeing the regex, and thinking it would pull the version out of `__init__.py`. I just threw this file in my repo to allow readthedocs to build the package.